Man page not being built leads to failing install of x42-plugins 20220913

Hi! When trying to install x42-plugins 20220913 I ran into an issue where the man page of this project is not built and therefore fails the installation of the plugins.

```
install -d /build/x42-plugins/pkg/x42-plugins/
make[1]: [Makefile:233: submodule_check] Error 1 (ignored)
install -d /build/x42-plugins/pkg/x42-plugins//usr/share/man/man1
install -m644 x42-spectr.1 /build/x42-plugins/pkg/x42-plugins//usr/share/man/man1
install -d /build/x42-plugins/pkg/x42-plugins//usr/lib/lv2/mixtri.lv2
make[1]: [Makefile:244: submodule_check] Error 1 (ignored)
install -d /build/x42-plugins/pkg/x42-plugins//usr/lib/lv2/fil4.lv2
install -d /build/x42-plugins/pkg/x42-plugins//usr/share/man/man1
install -d /build/x42-plugins/pkg/x42-plugins//usr/bin
install -m755 build/matrixmixerUI_gl.so /build/x42-plugins/pkg/x42-plugins//usr/lib/lv2/matrixmixer.lv2
install -m644 x42-phaserotate.1 /build/x42-plugins/pkg/x42-plugins/
install -m644 x42-fat1.1 /build/x42-plugins/pkg/x42-plugins//usr/share/man/man1
install -m755 build/sisco.so /build/x42-plugins/pkg/x42-plugins//usr/lib/lv2/sisco.lv2
install -d /build/x42-plugins/pkg/x42-plugins//usr/lib/lv2/spectra.lv2
install -m644 build/manifest.ttl build/mixtri.ttl /build/x42-plugins/pkg/x42-plugins//usr/lib/lv2/mixtri.lv2
install -m644 x42-tuna.1 /build/x42-plugins/pkg/x42-plugins//usr/share/man/man1
install: cannot stat 'x42-phaserotate.1': No such file or directory
install -m755 x42/x42-darc /build/x42-plugins/pkg/x42-plugins//usr/bin
install -m644 build/manifest.ttl build/fil4.ttl /build/x42-plugins/pkg/x42-plugins//usr/lib/lv2/fil4.lv2
make[1]: *** [Makefile:290: install-man] Error 1
make[1]: Leaving directory '/build/x42-plugins/src/x42-plugins-20220913/phaserotate.lv2'
install -d /build/x42-plugins/pkg/x42-plugins//usr/bin
install -m644 build/manifest.ttl build/spectra.ttl /build/x42-plugins/pkg/x42-plugins//usr/lib/lv2/spectra.lv2
install -d /build/x42-plugins/pkg/x42-plugins//usr/lib/lv2/fat1.lv2
make: *** [Makefile:40: phaserotate.lv2] Error 2
make: *** Waiting for unfinished jobs....
```

--- a/Makefile
+++ b/Makefile
@@ -203,7 +203,7 @@ submodule_check:
 submodules:
 	-test -d .git -a .gitmodules -a -f Makefile.git && $(MAKE) -f Makefile.git submodules
 
-all: submodule_check $(BUILDDIR)manifest.ttl $(BUILDDIR)$(LV2NAME).ttl $(targets) $(JACKAPP)
+all: submodule_check $(BUILDDIR)manifest.ttl $(BUILDDIR)$(LV2NAME).ttl $(targets) $(JACKAPP) man
 
 $(BUILDDIR)manifest.ttl: lv2ttl/manifest.ttl.in lv2ttl/manifest.gui.in Makefile
 	@mkdir -p $(BUILDDIR)

